### ISSUE TITLE
Files are not being made in save folder

### ISSUE THREAD

**Issue by KamelEbest** [dmustanger/7dtd-ServerTools#143]
I put the files in the Mods Folder, when i start the server it does not make the files that are supposed to be in the "save" folder.

This is happening to more people not just me, and no one has made an issue about it yet, therefor i made one.

**Comment by stevotvr** [comment]
It seems like `GamePrefs.GetString(EnumGamePrefs.SaveGameFolder)` doesn't always produce the expected value since the latest update. I guess it depends on how the setting is configured. I was able to work around this in my mod using `Path.GetFullPath($"{GameUtils.GetSaveGameDir()}{Path.DirectorySeparatorChar}..{Path.DirectorySeparatorChar}..")` instead.

**Comment by KamelEbest** [comment]
I have no ide what you are talking about stevotvr, i am used to drag and drop and it works, then config and done. I hope there wil be a fix for it. I am reading on alot of forums about people having the same issue.

**Comment by stevotvr** [comment]
Sorry, my response was mostly directed at the developers. Basically, the method this plugin uses to find the save directory is broken since the last update. If you set the SaveGameFolder setting to an absolute directory (if allowed by your host), it might work.

**Comment by KamelEbest** [comment]
Where can i change the save directory?

**Comment by KamelEbest** [comment]
This is the line now:

"<property name="SaveGameFolder"      value="c:\TCAFiles\Users\st****1\7**7\7dtdsaves"/>	"

**Comment by KamelEbest** [comment]
This is how it looks now:
"c:\TCAFiles\Users\s****m1\7***\7dtdsaves"

**Comment by KamelEbest** [comment]
...c:\TCAFiles\Users\stianm1\7837\7dtdsaves" : use this to override the default save game path...

**Comment by KamelEbest** [comment]
...c:\TCAFiles\Users\s****m1\7***\7dtdsaves" : use this to override the default save game path...

**Comment by stevotvr** [comment]
If it looks like the below, then it is already set to an absolute directory, so I don't have any more suggestions.
``
<property name="SaveGameFolder"      value="c:\TCAFiles\Users\stianm1\7837\7dtdsaves" />	
``

**Comment by KamelEbest** [comment]
yes it does. hmm i have no clue what to do then. hopefully the developers can make a fix.

**Comment by stevotvr** [comment]
I changed the line in the code and compiled it. You can try this version: [7dtd-ServerTools-test1.zip](https://github.com/dmustanger/7dtd-ServerTools/files/2678237/7dtd-ServerTools-test1.zip)

**Comment by KamelEbest** [comment]
OMG! thank you so much! that did the trick :D works now !

**Comment by stevotvr** [comment]
Great! I submitted my fix in a pull request.

#144

**Comment by KamelEbest** [comment]
You are the best! everything is ok and working like it should. this made my day! Thank you so much!
